preprocess.py

```diff
@@ -0,0 +1,176 @@
+from FixNamesTransformer import FixNamesTransformer
+# from DropSmallMissingData import DropSmallMissingData
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import Pipeline
+import pandas as pd
+import numpy as np
+import re
+from sklearn.preprocessing import OrdinalEncoder, OneHotEncoder, FunctionTransformer
+from sklearn.linear_model import LinearRegression
+from sklearn.experimental import enable_iterative_imputer  # IterativeImputer doesn't work without this import
+from sklearn.impute import IterativeImputer
+from feature_engine.selection import DropFeatures
+from typing import List, Tuple
+import warnings
+warnings.filterwarnings("ignore")
+
+
+def drop_small_missing_data(df: pd.DataFrame, features: List[str]) -> pd.DataFrame:
```

Comment:
Add doc (goes for all functions in this script, it doesn't need to be long, just a short sentence about what does the function do)